src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -1,31 +1,33 @@
-import { $$, Dom } from '../../utils/Dom';
+import 'styling/_ResponsiveTabs';
+import { reject, each, last } from 'underscore';
 import { InitializationEvents } from '../../events/InitializationEvents';
-import { PopupUtils, PopupHorizontalAlignment, PopupVerticalAlignment } from '../../utils/PopupUtils';
+import { Logger } from '../../misc/Logger';
+import { l } from '../../strings/Strings';
+import { $$, Dom } from '../../utils/Dom';
 import { EventsUtils } from '../../utils/EventsUtils';
+import { PopupHorizontalAlignment, PopupUtils, PopupVerticalAlignment } from '../../utils/PopupUtils';
+import { SVGDom } from '../../utils/SVGDom';
+import { SVGIcons } from '../../utils/SVGIcons';
 import { Utils } from '../../utils/Utils';
-import { Logger } from '../../misc/Logger';
 import { Component } from '../Base/Component';
 import { SearchInterface } from '../SearchInterface/SearchInterface';
 import { Tab } from '../Tab/Tab';
-import { IResponsiveComponent, ResponsiveComponentsManager, IResponsiveComponentOptions } from './ResponsiveComponentsManager';
-import { ResponsiveComponentsUtils } from './ResponsiveComponentsUtils';
-import { l } from '../../strings/Strings';
 import { ResponsiveComponents } from './ResponsiveComponents';
-import * as _ from 'underscore';
-
-import 'styling/_ResponsiveTabs';
-import { SVGIcons } from '../../utils/SVGIcons';
-import { SVGDom } from '../../utils/SVGDom';
+import { IResponsiveComponent, IResponsiveComponentOptions, ResponsiveComponentsManager } from './ResponsiveComponentsManager';
+import { ResponsiveComponentsUtils } from './ResponsiveComponentsUtils';
 
 export class ResponsiveTabs implements IResponsiveComponent {
   private static DROPDOWN_HEADER_LABEL_DEFAULT_VALUE = 'More';
+  private static TAB_IN_DROPDOWN_CSS_CLASS = 'coveo-tab-dropdown';
+  private static TAB_IN_DROPDOWN_HEADER_CSS_CLASS = `${ResponsiveTabs.TAB_IN_DROPDOWN_CSS_CLASS}-header`;
   private static logger: Logger;
   private dropdownHeader: Dom;
   private dropdownContent: Dom;
   private tabSection: Dom;
   private documentClickListener: EventListener;
   private searchInterface: SearchInterface;
   private dropdownHeaderLabel: string;
+  private _initialTabsOrder: HTMLElement[];
```

Comment:
Why the `_` here, to indicate that's it's private. If so why only this private variable?